### DISCUSSION TITLE
2.12.6-SNAPSHOT

### DISCUSSION THREAD

**Discussion by nikosgram** [nikosgram/gringotts#157] [Announcements]
## What's Changed
* Fix NPE related to shulker boxes by @GroobleDierne in https://github.com/nikosgram/gringotts/pull/155

**Full Changelog**: https://github.com/nikosgram/gringotts/compare/2.12.5...2.12.6-SNAPSHOT

<hr /><em>This discussion was created from the release <a href='https://github.com/nikosgram/gringotts/releases/tag/2.12.6-SNAPSHOT'>2.12.6-SNAPSHOT</a>.</em>

**Comment by EmilPetrakiev** [comment]
Is the plugin alive and maintained? I saw that the plugin was not updated from 2021 and it got an update now. Is the plugin going to continue to be maintained?

**Reply by nikosgram** [reply]
It is, although less active. Me and the community we are trying our best to keep up with bug fixes and new versions of Minecraft :) 

We have stopped releasing on bukkit.dev, and we are using only spigotmc.org for our releases.

**Reply by EmilPetrakiev** [reply]
Thanks for the confirmation!
